src/main/java/seedu/address/model/person/RoleContainsKeywordsPredicate.java

```diff
@@ -5,6 +5,11 @@
 
 import seedu.address.commons.util.StringUtil;
 
+/**
+ * Tests that a {@code Person}'s {@code Role} matches all the keywords given.
+ *
```

Comment:
Not sure if empty line in javadoc separates two sentences, but you may want to use `<p>` to write paragraphs for description as explained here on [checkstyle website](https://checkstyle.sourceforge.io/config_javadoc.html#JavadocParagraph)

Reply:
Thanks for the suggestion. I am not sure either so I will make the necessary changes